packages/material/test/renderers/MaterialArrayLayout.test.tsx

```diff
@@ -154,49 +157,52 @@ describe('Material array layout', () => {
 
   it('should render two by two children', () => {
     const store = initJsonFormsStore();
-    const tree = TestUtils.renderIntoDocument(
+    const wrapper = mount(
       <Provider store={store}>
         <MaterialArrayLayout
           schema={schema}
           uischema={uischema}
         />
       </Provider>
-    ) as React.Component<any, any, any>;
+    );
 
-    const controls = TestUtils.scryRenderedDOMComponentsWithTag(tree, 'input');
+    const controls = wrapper.find('input');
     // 2 data entries with each having 2 controls
-    expect(controls.length).toBe(4);
+    expect(controls).toHaveLength(4);
+    wrapper.unmount();
```

Comment:
why not use `afterEach(() => wrapper.unmount());` once as in the arraycontrol test?